packages/mdc-list/mdc-list.scss

```diff
@@ -21,26 +21,12 @@
 @import "./mixins";
 @import "./variables";
 
-.mdc-list,
```

Comment:
I'm pretty sure you can remove the import for `@material/theme/mixins` after removing the dark styles in this file? (We also missed this for mdc-button.scss I think, so might want to send a tiny separate PR for that, and check on the other open PRs)

Reply:
Will do...good call.